forui/lib/src/widgets/alert/alert.dart

```diff
@@ -186,16 +193,24 @@ final class FAlertCustomStyle extends FAlertStyle with Diagnosticable {
       decoration.hashCode ^ padding.hashCode ^ icon.hashCode ^ titleTextStyle.hashCode ^ subtitleTextStyle.hashCode;
 }
 
-class _InheritedData extends InheritedWidget {
+@internal
```

Comment:
Why is this `@internal`?

Reply:
Ah we don’t expose this one since there isn’t a raw constructor that can benefit from it.